ql/src/test/org/apache/hadoop/hive/ql/TestTxnNoBuckets.java

```diff
@@ -833,10 +833,6 @@ public void testCompactStatsGather() throws Exception {
     Assert.assertEquals("Unexpected number of compactions in history", 1, resp.getCompactsSize());
     Assert.assertEquals("Unexpected 0 compaction state", TxnStore.CLEANING_RESPONSE, resp.getCompacts().get(0).getState());
     Assert.assertTrue(resp.getCompacts().get(0).getHadoopJobId().startsWith("job_local"));
-
-    //now check that stats were updated
-    map = hms.getPartitionColumnStatistics("default","T", partNames, colNames, Constants.HIVE_ENGINE);
```

Comment:
should we check basic statistics here instead on colStat?

Reply:
Modified the test to check basic stats only